mail_tracking/models/mail_tracking_email.py

```diff
@@ -473,3 +473,41 @@ def event_process(self, request, post, metadata, event_type=None):
         # - return 'NONE' if this request is not for you
         # - return 'ERROR' if any error
         return "NONE"  # pragma: no cover
+
+    def _get_old_mail_tracking_email_domain(self, max_age_days):
+        target_write_date = fields.Datetime.subtract(
+            fields.Datetime.now(), days=max_age_days
+        )
+        return [("write_date", "<", target_write_date)]
+
+    @api.autovacuum
+    def _gc_mail_tracking_email(self, limit=5000):
+        config_max_age_days = (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("mail_tracking.mail_tracking_email_max_age_days")
+        )
+        try:
+            max_age_days = int(config_max_age_days)
+        except ValueError:
+            max_age_days = 0
+
+        if not max_age_days > 0:
+            _logger.info("Mail tracking email deletion is disabled.")
```

Comment:
As the autovacuum is run quite often, I would switch this log to DEBUG:
```suggestion
            _logger.debug("Mail tracking email deletion is disabled.")
```

Reply:
I don't see this as needed really. It will spam the log with no reason.

Reply:
Good points, removed it